cfg/validate.go

```diff
@@ -72,6 +72,13 @@ func IsValidExperimentalMetadataPrefetchOnMount(mode string) error {
 	}
 }
 
+func isValidSequentialReadSizeMB(sequentialReadSizeMB int64) error {
```

Comment:
nit: Avoid repetition - https://google.github.io/styleguide/go/decisions#repetition
```suggestion
func isValidSequentialReadSizeMB(size int64) error {
```